src/test/py/base.py

```diff
@@ -267,7 +344,7 @@ def clean_db(request):
     """
     pdb = request.module.pipeline
     request.addfinalizer(pdb.drop_all_queries)
-
+    pdb.execute("select pipeline_trigger_debug('setup')")
```

Comment:
This is a replacement for trigger_testing_setup - it clears the shared file that is being written to by test_alert_new_row. I will move it into a specific fixture.